monitor/data_monitoring.py

```diff
@@ -30,6 +30,7 @@ def __init__(self, config: Config, force_update_dbt_package: bool = False,
         self.execution_properties = {}
         self.slack_webhook = slack_webhook or self.config.slack_notification_webhook
         self._download_dbt_package_if_needed(force_update_dbt_package)
+        self.status_code = 0
```

Comment:
Well I like being more specific.
status code is used for? 0 and 1 are numbers with no real meaning.
IMO it is better to call it run_successfully and set it t to True/False.
It much more explicit :) 
WDYT?